### ISSUE TITLE
Can't type password after resume

### ISSUE THREAD

**Issue by kyoh86** [kyoh86/dotfiles#32]
referenced issues:
[No keyboard focus in unlock dialog XFCE - Ask Ubuntu](https://askubuntu.com/questions/1131939/no-keyboard-focus-in-unlock-dialog-xfce)
[Bug #1824322 “Unlock dialog does not have keyboard focus” : Bugs : lightdm-gtk-greeter package : Ubuntu](https://bugs.launchpad.net/ubuntu/+source/lightdm-gtk-greeter/+bug/1824322)
[Bug #1311403 “Can't type password after resume.” : Bugs : unity package : Ubuntu](https://bugs.launchpad.net/ubuntu/+source/unity/+bug/1311403)
[1323948 – After locking/unlocking screen with light-locker, Xfce environment pointer is missing/invisible](https://bugzilla.redhat.com/show_bug.cgi?id=1323948)

soved hint:
[Screensaver issue with XFCE 4.14 / Desktop / Xfce Forums](https://forum.xfce.org/viewtopic.php?pid=54966#p54966)

> I have the same problem in OpenBSD 6.6. Uninstalling xscreensaver resolved the problem for now.
